File: Sources/SimERP/ClientApp/src/app/lists/currency/model/currency.ts
export class Currency {
  CurrencyId = '';
  CurrencyName: string;
  Notes: string;
  CreatedBy: number;
  CreatedDate: Date;
  ModifyBy: number;
  ModifyDate: Date;
  SearchString: string;
  SortOrder: number;
  IsMainCurrency: boolean;
  IsActive: boolean;

  // custom propeties
  UserName: string;
}

File: Sources/SimERP.Data/DBEntities/Currency.cs
using System;
using System.Collections.Generic;

namespace SimERP.Data.DBEntities
{
    public partial class Currency
    {
        public string CurrencyId { get; set; }
        public string CurrencyName { get; set; }
        public string Notes { get; set; }
        public int CreatedBy { get; set; }
        public DateTimeOffset CreatedDate { get; set; }
        public int? ModifyBy { get; set; }
        public DateTimeOffset? ModifyDate { get; set; }
        public string SearchString { get; set; }
        public int SortOrder { get; set; }
        public bool IsMainCurrency { get; set; }
        public bool? IsActive { get; set; }
    }
}
